DependencyInjection/FMElfinderExtension.php

```diff
@@ -29,4 +29,9 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('fm_elfinder', $config);
         $container->setAlias('fm_elfinder.configurator', $config['configuration_provider']);
     }
+
+    public function getNamespace()
+    {
+        return 'http://helios-ag.github.io/schema/dic/fm_elfinder';
```

Comment:
a namespace is an unique identifier for elements belonging to your bundle. By convention, this is a URL (which doesn't have to exists). The Symfony convention is to use `PROJECT_SITE/schema/dic/DI_ALIAS`.